views.py

```diff
@@ -0,0 +1,64 @@
+from app import app
+from flask import render_template,request,redirect,jsonify,make_response
+from flask import request, redirect
+from werkzeug.utils import secure_filename
+import os
+
+@app.route("/")
+def index():
+	return render_template("public/index.html")
+
+@app.route("/json_check", methods=["POST"])
```

Comment:
why is this a seperate endpoint, i don't think we'll be exposing this, consider making this a method

Reply:
removed